omnibus_overrides.rb

```diff
@@ -2,8 +2,8 @@
 # When updating this, check doc/FrequentTasks.md for checklists to ensure all
 # the various usages are updated in lockstep
 #
-override :rabbitmq, version: "3.6.15"
-override :erlang, version: "20.3.8.9"
+override :rabbitmq, version: "3.7.18"
```

Comment:
Do we need this line at all anymore since we no longer ship rabbitmq?

Reply:
We probably don't need it anymore.  That change was from before we deleted rabbit.

Reply:
Will delete and see what happens.

Reply:
Fixed.